packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -1273,6 +1273,12 @@ export async function putOrganization(
       }
     }
 
+    if (licenseKey && licenseKey.trim() !== org.licenseKey) {
+      updates.licenseKey = licenseKey.trim();
+      orig.licenseKey = org.licenseKey;
+      await setLicenseKey(org, updates.licenseKey);
```

Comment:
This is calling `updateOrganization` immediately before another `updateOrganization` call.   Doesn't quite seem like the right level of abstraction.

Reply:
setLicenseKey does not call updateOrganization, it just sets org.licenseKey to be the new key and verifies it is a valid key.